docker/validate_docs.sh

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env bash
```

Comment:
This is the script that ensures each example in the docs can be run using python.